src/main/java/com/funkydonkies/camdetect/VideoCap.java

```diff
@@ -15,7 +15,7 @@
 
     private VideoCapture cap;
     private Mat2Image mat2Img = new Mat2Image();
-    private static final int INPUTSOURCENUMBER = 0;
+    private static final int INPUTSOURCENUMBER = 2;
```

Comment:
Should be divided by underscores, SOURCE_INPUT_NUMBER also makes more sense to me